.circleci/config.yml

```diff
@@ -278,17 +272,13 @@ commands:
           command: docker login -u "$QUAY_USER" -p $QUAY_PASS quay.io
 
   bail_if_forked:
-    parameters:
-      exit_code:
-        type: integer
-        default: 1
     steps:
       - run:
           name: Exit if running from a fork
           command: |
             if [ -n "$CIRCLE_PR_NUMBER" ]; then
               echo 'Error: Running from a fork, exiting!'
-              exit << parameters.exit_code >>
+              exit 1
```

Comment:
Does this mean the new `e2e` job is guaranteed to always fail on PRs from forks, until we do the git work to push the commits into the main repository?

Reply:
I think it depends on what we want the experience to be for external PRs. If we want to require e2e to run for everything, having the job fail until we take an action makes sense (though it'd be nice if we could automatically post some comment to the PR explaining what's happening). If we're ok having e2e be optional, I think there's a way to conditionally skip the e2e job at the workflow-definition level:
```
filters:
  branches:
    ignore: /pull\/[0-9]+/
```